packages/plugin/src/playerContribPlugin.tsx

```diff
@@ -155,6 +158,9 @@ export abstract class PlayerContribPlugin extends (KalturaPlayer as any).core.Ba
                 serviceUrl: this.player.config.provider.env.serviceUrl,
                 partnerId: this.player.config.session.partnerId,
                 userId: this.player.config.session.userId
+            },
+            kitchenSink: {
```

Comment:
the player contrib plugin should not be aware to ui stuff.. what did you try to resolve by this?